packages/fungible-token/bridge-fungible-token/tests/functions/src20/mod.rs

```diff
@@ -0,0 +1,98 @@
+mod success {
+    use crate::utils::{
+        constants::{
+            BRIDGED_TOKEN, BRIDGED_TOKEN_DECIMALS, BRIDGED_TOKEN_ID, FROM, PROXY_TOKEN_DECIMALS,
+        },
+        interface::src20::{decimals, name, symbol, total_supply},
+        setup::{
+            create_msg_data, create_wallet, get_asset_id, relay_message_to_contract,
+            setup_environment, BridgeFungibleTokenContract, BridgingConfig,
+        },
+    };
+    use fuels::accounts::wallet::WalletUnlocked;
+
+    /// This setup mints tokens so that they are registered as minted assets in the bridge
+    async fn setup_test() -> BridgeFungibleTokenContract<WalletUnlocked> {
+        let mut wallet = create_wallet();
+
+        let config = BridgingConfig::new(BRIDGED_TOKEN_DECIMALS, PROXY_TOKEN_DECIMALS);
+
+        let (message, coin, deposit_contract) = create_msg_data(
+            BRIDGED_TOKEN,
+            BRIDGED_TOKEN_ID,
+            FROM,
+            *wallet.address().hash(),
+            config.amount.test,
+            None,
+            false,
+            None,
+        )
+        .await;
+
+        let (contract, utxo_inputs, _) = setup_environment(
+            &mut wallet,
+            vec![coin],
+            vec![message],
+            deposit_contract,
+            None,
+            None,
+        )
+        .await;
+
+        let _receipts = relay_message_to_contract(
+            &wallet,
+            utxo_inputs.message[0].clone(),
+            utxo_inputs.contract,
+            &utxo_inputs.coin[..],
+        )
+        .await;
+
+        contract
+    }
+
+    #[ignore]
```

Comment:
This test is ignored? 
It looks like the comment about the SDK is outdated and so this test can be un-ignored and the assertion added back in?

Reply:
Fixed